create_db.py

```diff
@@ -1,30 +1,21 @@
 # standard libraries
-import json, logging, os
-
-# third-party libraries
-from sqlalchemy import create_engine
+import logging
 
 # local libraries
 from db.db_creator import DBCreator
-
+from environ import ENV
 
 # Initializing settings and global variables
 
 logger = logging.getLogger(__name__)
 
-try:
-    config_path = os.getenv("ENV_PATH", os.path.join('config', 'secrets', 'env.json'))
-    with open(config_path) as env_file:
-        ENV = json.loads(env_file.read())
-except FileNotFoundError:
-    logger.error('Configuration file could not be found; please add env.json to the config directory.')
-
 DB_PARAMS = ENV['INVENTORY_DB']
+APPEND_TABLE_NAMES = ENV.get('APPEND_TABLE_NAMES', ['job_run'])
```

Comment:
I feel like we should have DEFAULT_APPEND_TABLES that we don't have to define in the config. Are we ever going to want these dropped automatically by the script?

like 
DEFAULT_APPEND_TABLES = ['job_run', "data_source_status"]
APPEND_TABLE_NAMES = ENV.get('APPEND_TABLE_NAMES', []).extend(DEFAULT_APPEND_TABLES)

Reply:
I created an issue, #66.